src/core/Dropdown/Dropdown/Dropdown.baseStyles.tsx

```diff
@@ -1,110 +1,145 @@
 import { css } from 'styled-components';
 import { SuomifiTheme } from '../../theme';
-import { element, inputButton } from '../../theme/reset';
+import { element, input } from '../../theme/reset';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
+  width: 290px;
+  /* stylelint-disable no-descending-specificity */
```

Comment:
Does this need to be disabled or styles moved accordingly?

Reply:
I couldn't get all the styles working and the linter to be happy at the same time. 

This linter rule has been problematic in some other components as well